containerd-shim/process.go

```diff
@@ -132,7 +132,8 @@ func (p *process) start() error {
 		)
 	} else if p.checkpoint != nil {
 		args = append(args, "restore",
-			"--image-path", filepath.Join(p.checkpointPath),
+			"--image-path", p.checkpointPath,
+			"--work-path", filepath.Join(p.checkpointPath, "criu.work", "restore-"+time.Now().Format(time.RFC3339)),
```

Comment:
This may be a dumb question, but why do we need to  append the time there? A new directory is created every time a restore is done? When do they get deleted?